src/main/resources/com/mathworks/ci/tools/MatlabInstaller/help-products.html

```diff
@@ -0,0 +1,13 @@
+<div>
+  <p>
+    Insert a space-separated list of products to install. The plugin installs the specified products in addition to MATLAB.
+  </p>
+  <p>
+    For a list of supported products, open the input file for your preferred release from the <a href="https://github.com/mathworks-ref-arch/matlab-dockerfile/tree/main/mpm-input-files">mpm-input-files </a>folder on GitHub.
```

Comment:
files </a>folder   --> files</a> folder

Reply:
could not get the change did you mean `open the input file folder` instead `open the input file` ?

Reply:
I meant there is a white space before the </a> tag. I think that white space should be after the tag instead.

Reply:
Done